EmpleoDotNet.Core/Domain/JobOpportunityLocation.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace EmpleoDotNet.Core.Domain
+{
+    public class JobOpportunityLocation
+    {
+        public int Id { get; set; }
+        public string PlaceId { get; set; }
+        public string Name { get; set; }
+        public string Latitude { get; set; }
+        public string Longitude { get; set; }
+    }
+}
```

Comment:
This name implies the Location entity is necessarily tied (harcoded) to the JobOpportunity which is not the case. Even though JobOpportunity uses it, it doesnt mean other entities can't (potentially) use it.